trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/testing/TestingUtil.java

```diff
@@ -148,4 +154,13 @@ public static String sha256(String content)
     {
         return Hashing.sha256().newHasher().putString(content, StandardCharsets.UTF_8).hash().toString();
     }
+
+    public static TestingHttpServer createTestingHttpCredentialsServer(Servlet servlet)
```

Comment:
This is not specific to credentials.

We could also wrap the `IOException` into a `RuntimeException` (which the caller does now)

```suggestion
    public static TestingHttpServer createTestingHttpServer(Servlet servlet)
```

Reply:
Sorry, my bad. I'll change the name. 
I think its better to throw the original exception than a transformed exception in this case since its a generic function

Reply:
Its a testing function, doesn't make much difference, I'm fine with both